frontend/src/components/dropdown/SelectDropdown.tsx

```
@@ -36,17 +37,18 @@ const SelectDropdown: React.FC<SelectDropdownProps> = ({
   placeholder,
   requiredAlert = false,
   requiredField = false,
+  disabled = false,
 }) => {
   return (
-    <Field aria-required={requiredField}>
+    <Field aria-required={requiredField} disabled={disabled}>
       <Label className="block text-sm font-medium leading-6 mb-2 text-gray-300">
         {title + ' '}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)